app/src/main/res/values/strings.xml

```diff
@@ -349,6 +349,11 @@
     <string name="tx_status_failed">Failed</string>
     <string name="tx_status_success">Success</string>
 
+    <string name="tx_submitted">Your transaction is submitted!</string>
+    <string name="tx_submitted_desc">It normally takes some time for a transaction to be executed.</string>
```

Comment:
This is probably the text from the specification. But it sounds like no further action is necessary. Shouldn't we tell the user that it needs to executed on web or ios?

Reply:
@DmitryBespalov @TanyaEfremova wdyt?

Reply:
This is what used by iOS:

```swift
        let title = "Your transaction is queued!"
        let body = "Your request to send \(formattedAmount) \(token) is submitted and needs to be confirmed by other owners."
```